health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/CampaignBoundary.js

```
@@ -0,0 +1,811 @@
+    const [statusMap, setStatusMap] = useState({});
+    useEffect(() => {
+        if (selectedData && selectedData.length >= 0) {
+            setStatusMap(() => Digit.Utils.microplanv1.createStatusMap(selectedData, boundaryHierarchy))
+        }
+    }, [boundaryHierarchy])
+
+    // Prevent unnecessary useEffect calls
+    useEffect(() => {
+        if (!boundaryHierarchy || boundaryHierarchy.length === 0) return;
+
+        let bH = [];
```

Biome

[error] This let declares a variable that is only assigned once.

'bH' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)